pad/hori/Fighting Stick α/config.json

```diff
@@ -0,0 +1,94 @@
+{
+    "name": "Hori Fighting Stick α for PlayStation 5",
+    "functions": {
+        "analog_sticks": null,
+        "digital_sticks": null,
+        "action_buttons": [
+            {
+                "name": "Square",
+                "analog": false
+            },
+            {
+                "name": "Circle",
+                "analog": false
+            },
+            {
+                "name": "Cross",
+                "analog": false
+            },
+            {
+                "name": "Triangle",
+                "analog": false
+            },
+            {
+                "name": "L1",
+                "analog": false
+            },
+            {
+                "name": "R1",
+                "analog": false
+            },
+            {
+                "name": "L2",
+                "analog": false
+            },
+            {
+                "name": "R2",
+                "analog": false
+            },
+            {
+                "name": "L3",
+                "analog": false
+            },
+            {
+                "name": "R3",
+                "analog": false
+            }
+        ],
+        "menu_buttons": ["Options"],
+        "system_buttons": ["Create", "PS Button"],
+        "other_buttons": ["Function", "Tournament Lock"],
+        "trackpads": [ 
+            {
+            "name": "Trackpad",
+            "clickable": true
+            }
+        ],
+        "touch_screens": null,
+        "directional_buttons": [
```

Comment:
In the case of a fight stick, directions are handled by the "Digital Stick" element (See https://github.com/Wolfy76700/ControllerDatabase/wiki/Controller-Definition#functions_digital_sticks_items_name)